changelogs/aws.yaml

```diff
@@ -0,0 +1,75 @@
+- changelogs:
+  - component: chart-operator
```

Comment:
To order all changelogs based on `component`, I put the component name as the first key in this array.

Reply:
I think the file will get big so it will be easier to navigate if the changelogs are grouped by release. Something like this.

```
- release: 10.1.1
  - apps:
    - app: chart-operator
      - version: 0.11.2
        description: Remove CPU Limits.
        type: Changed
        urls:
          - https://github.com/giantswarm/chart-operator/pull/335
    - app: coredns
      - version: 1.1.3
        componentVersion: 1.6.5
        description: Fix HPA manifests for Kubernetes 1.16.
        type: Changed
        urls:
          - https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v113-2020-01-08
```

WDYT?

Reply:
big +1

Reply:
Is this a valid yaml? I think the value can be either string or list.

```
- release: 10.1.1
  - apps:
    - app: chart-operator
```

Reply:
I would just bring the version as a first key here as a convention. Sorting this alphabetically isn't really helpful.

Reply:
@kopiczko I'm not sure what the first key as you suggest, would you mind to give me an example?

Reply:
@tomahawk28 sorry I confused things a little with the `release:` key being in alphabetical order. 

But still this is not a valid yaml:

```
  - release: 10.1.1
    - component: chart-operator
```

This is:

```
  - release-10.1.1:
    - component: chart-operator
```

Reply:
Yes sorry I did my suggestion too fast. I like this. It's very human readable but should also be easy to parse in the cluster-operator code.

Reply:
Yeah this is a general problem with dynamic keys.

I think this is also valid:

```
- "10.1.1":
  - component: chart-operator
```

Reply:
Do we really need a double quote? I tried a few validators online but it doesn't complain about that.

Reply:
Maybe we don't.